### DISCUSSION TITLE
VS Olaia FULL WEEK

### DISCUSSION THREAD

**Discussion by gunkilock** [KadeArchive/Kade-Engine#888] [Show and tell]
Is a mod i make  enjoy the mod!
https://gamebanana.com/mods/292744
(sorry for bad english im spanish lol)

**Comment by ACardboardBox1** [comment]
thank you, but not the place to do it 
pls close

**Comment by gunkilock** [comment]
Ok im sorry

**Comment by Lucky-56** [comment]
I think you may put this in discussion with the type of 'Show and Tell'.
If not well then what is that for
